### PR TITLE
docs: update faq to correctly address firefox and bidi

### LINKED ISSUE #12769: [Docs]: Update FAQ page for Puppeteer v23
### Feature description

Related to https://github.com/puppeteer/puppeteer/issues/12728.

The FAQ page mentions experimental support for both BiDi and Firefox and needs to be updated with v23, ideally also answering the question where CDP will go (Answer: nowhere, things will continue as is).

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Fixes #12769.

This updates some of the language in the FAQ to correctly address support of both Chrome and Firefox. It also adds a new question "What will happen to CDP?" and also removes some of the mentions that hint at browser support expanding in the future.

### CODE REVIEW

**Comment by OrKoN** [comment]
@matthiasrohmer could you please rebase?

File: docs/faq.md [hunk 1]
```diff
@@ -8,34 +8,25 @@ expertise on the project! See our
...
-to automate Firefox. The WebDriver BiDi implementation in Firefox will replace
-the current CDP implementation in Firefox in the future. See
-https://pptr.dev/webdriver-bidi for more details.
+From Puppeteer v23.0.0 onwards Puppeteer provides support for both Chrome and Firefox.
+
+To automate Chrome Puppeteer uses the Chrome DevTools Protocol (CDP) by default, but it can
+also be automated using WebDriver BiDi which is the default for automating Firefox.
+
+To understand the subtle differences in API support refer to our
+[WebDriver BiDi guide](https://pptr.dev/contributing).
```
Comment (OrKoN) [inline docs/faq.md]: wrong link?

Comment (matthiasrohmer) [reply docs/faq.md]: Yes, thanks for spotting! Fixed.

**Comment by matthiasrohmer** [comment]
Done! My fork got out of sync.

**Comment by OrKoN** [comment]
could you please reformat? `npm run format`

**Review by OrKoN — APPROVED** [review]

**Comment by matthiasrohmer** [comment]
Also done, thank you!

**Pull request merged**